examples/example-react-native-wallet/index.js

```diff
@@ -6,13 +6,19 @@ import 'react-native-get-random-values'
 import {AppRegistry} from 'react-native';
 import App from './App';
 import MobileWalletAdapterEntrypointBottomSheet from './screens/MobileWalletAdapterEntrypointBottomSheet';
+import TestingEntrypointBottomSheet from './screens/TestingEntrypointBottomSheet';
```

Comment:
should we delete the testing bottom sheet now that everything is hooked up? wonder if its worth keeping for development, tho its also another thing to maintain.

Reply:
keeping for now, its an example app after all